### DISCUSSION TITLE
Practica de Pastelería con VUE

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#559] [Show and tell]
Hola

Recien termine la practica de VUE, si gustan checarla es esta:
https://oscardanielm.github.io/DeployPasteleria/

El código esta aquí:
https://github.com/oscardanielM/DeployPasteleria

Cualquier duda en lo que les pueda ayudar solo díganme

**Comment by RodolfoMorquecho** [comment]
Te quedo muy bien Oscar, le diste continuación a tu pagina de la practica 1 o cambiaste logos,colores,etc?

**Reply by oscardanielM** [reply]
Yes le di como una continuación, solo cambiaron un poco algunas cosas de diseño.

**Comment by luisthedataguy** [comment]
Que quedo muy bien bro. No tenía idea como hacerla, pero ya me di una idea. Muchas gracias.

**Reply by oscardanielM** [reply]
Va no hay de que, cualquier cosa que tengas duda dime ja si puedo apoyarte pues adelante

**Comment by RobertoPeredo** [comment]
Hola, me puedes ayudar. No sé cómo desplegar mi proyecto de vue en GitHub pages.

**Reply by oscardanielM** [reply]
Yes 

Lo primero que tienes que hacer es cambiar la configuración de ruta para notificar que vas a subir a un servidor, esa configuracion se hace en el archivo de vue.config.js

module.exports = defineConfig({
  publicPath: process.env.NODE_ENV === 'production'
  ? '/DeployPasteleria/'
  : '/'
})

En este caso puse el nombre de mi repositorio para que me redireccionara a mi sitio al momento de hacer el deploy, si no haces esto, al subir el deploy a git te mostrara un error **502** je

Ya después que hiciste el cambio en el vue.config.js tienes que generar el deploy
**npm run build**

Y listo te genera el deploy que es el que tendrás que subir a git y activar como github page

Si no me di a entender mucho ja esta pagina te puede ayudar un poco mas
https://cli.vuejs.org/guide/deployment.html#github-pages

**Reply by D-CIoud** [reply]
Buenas bro, crees que me puedas apoyar un poco?, realice algunos de los pasos del link que pusiste pero no me ha sido posible publicar el proyecto en GitHubPages :c

**Reply by oscardanielM** [reply]
Si dime, en que te puedo ayudar o que error es el que te marca

**Reply by D-CIoud** [reply]
Muchas gracias disculpa por la tardanza en contestar, es que mira ya hice el cambio en el vue.config.js y también generé un archivo deploy, pero no entiendo lo demás tengo que generar un archivo travis.yml para publicar ?, ya hice también el npm run build pero estoy perdido de ahí

**Reply by oscardanielM** [reply]
Ok, yo primero hice el cambio de mi vue.config.js y despues realice el deploy de ahí solo sube esos archivos a tu repositorio y ya te vas a configuracion -> page -> y de hay seleccionas la rama y ya le das en guardar.

Yo no necesite hacer algo mas je con eso funciono porque el problema que tenia era el 502, que me decía que no se podía publicar la url

**Comment by MagdielAndres** [comment]
Te quedo bien  tú página web

**Reply by oscardanielM** [reply]
Gracias gracias je

**Comment by SergioSaulAM** [comment]
No inventes Bro, te quedó super bien. Espero sigas así que eres un grande :D. Y también gracias por la ayuda a los que tuvimos dudas en cómo hacerlo.

**Reply by oscardanielM** [reply]
Va no hay de que ja para eso estamos para apoyar a los demás

**Comment by LuceroLuciano** [comment]
Explorer, tu pagina quedo increíble 100/100.
